business-registry-dashboard/app/components/BusinessLookup.vue

```diff
@@ -29,16 +29,19 @@ const searchType = ref<'reg' | 'namex'>('reg')
       >
         <template #empty>
           <div class="flex flex-col gap-2 px-4 py-2">
-            <span class="font-semibold">{{ $t(`search.${searchType}.empty.title`) }}</span>
-            <span>{{ $t(`search.${searchType}.empty.content`) }}</span>
+            <span class="font-semibold text-bcGovColor-midGray">{{ $t(`search.${searchType}.empty.title`) }}</span>
+            <span class="text-[#405057]">{{ $t(`search.${searchType}.empty.content`) }}</span>
```

Comment:
Can we get away from using hex colours in the template?

Reply:
Also on line 40.

Reply:
OK, and then keep a "to do" to update these in the code later. (tech debt)

Reply:
I don't have permission in this repo to resolve conversations like this one. Please resolve.

Reply:
Deal, will do. New ticket: https://app.zenhub.com/workspaces/entities---david-65af11a89e89f5043c2911f6/issues/gh/bcgov/entity/24400

Reply:
Awesome, thanks!